src/Hosting.CommandLine/HostBuilderExtensions.cs

```diff
@@ -28,12 +28,14 @@ public static class HostBuilderExtensions
         /// <typeparam name="TApp">The type of the command line application implementation</typeparam>
         /// <param name="hostBuilder">This instance</param>
         /// <param name="args">The command line arguments</param>
+        /// <param name="configure">The delegate to configure the application</param>
         /// <param name="cancellationToken">A cancellation token</param>
         /// <returns>A task whose result is the exit code of the application</returns>
         public static async Task<int> RunCommandLineApplicationAsync<TApp>(
-            this IHostBuilder hostBuilder, string[] args, CancellationToken cancellationToken = default)
+            this IHostBuilder hostBuilder, string[] args, Action<CommandLineApplication> configure = null, CancellationToken cancellationToken = default)
```

Comment:
This probably needs a separate overload so that this isn’t a breaking change.

Reply:
make sense